Changelog.md

```diff
@@ -6,6 +6,7 @@
 ### New rules
 
 * A label element may not have labelable descendants other than its labeled control (`src/audits/MultipleLabelableElementsPerLabel.js`)
+* A data table must identify row and column headers (`src/audits/TableHasAppropriateHeaders.js`)
```

Comment:
Sorry, because I took a while to get back to this, this Changelog line now needs to be moved above the last Release heading.